react/features/base/participants/functions.js

```diff
@@ -338,7 +338,7 @@ export function getRemoteParticipantsSorted(stateful: Object | Function) {
 export function getPinnedParticipant(stateful: Object | Function) {
     const state = toState(stateful);
     const { pinnedParticipant } = state['features/base/participants'];
-    const stageFilmstrip = isStageFilmstripEnabled(state);
+    const stageFilmstrip = shouldDisplayStageFilmstrip(state, 0);
 
     if (stageFilmstrip) {
         const { activeParticipants } = state['features/filmstrip'];
```

Comment:
I'm sorry for the dummy question but in the case when we have pinned many participants, what will this function return?

Reply:
This function is only used on stage filmstrip when there's only one participant (in order to determine which participant to display on large video)